src/UI/Badge.elm

```diff
@@ -4,6 +4,7 @@ module UI.Badge exposing
     , grayDark, primaryDark, successDark, warningDark, dangerDark
     , withTone
     , renderElement
+    , outlineDark, outlineLight, withIcon
```

Comment:
You need to add those somewhere in the @docs lines below...

Reply:
I've added for "withIcon". The other ones are already there ?

Reply:
Done